docs/modules/ROOT/nav-how-to-guides.adoc

```diff
@@ -9,6 +9,7 @@
 *** xref:how-to-guides/configuring-builds/proc_preventing_redundant_rebuilds.adoc[Preventing redundant rebuilds]
 ** Testing your application
 *** xref:how-to-guides/testing_applications/con_test-overview.adoc[Overview of {ProductName} tests]
+**** xref:how-to-guides/testing_applications/enable_snyk_check_for_a_product.adoc[Enable snyk task]
```

Comment:
Shouldn't it be under surface tests instead of overview?

Reply:
I feel [surface test](https://redhat-appstudio.github.io/docs.appstudio.io/Documentation/main/how-to-guides/testing_applications/surface-level_tests/) might be better, we may add a new section after the `Label checks` https://redhat-appstudio.github.io/docs.appstudio.io/Documentation/main/how-to-guides/testing_applications/surface-level_tests/#label-checks, maybe?

Reply:
Yeah right, the navigation page makes more sense under surface level test.

Reply:
I feel there should be a separate section for Product security tests where this Snyk doc would make more sense ideally. On the over view page there is a section for product security test we just dont have a details page for it. I will create a new story for this.

For now, added the Snyk nav page under surface level tests.

Reply:
Filed https://issues.redhat.com/browse/STONEINTG-766